index.rst

```diff
@@ -44,14 +44,579 @@
+  
+A solicitation for use-cases was made to the Rubin community via the Commissioning meeting and a general call on the Slack channel.
+A meeting was held to define what the group was looking for and provided examples.
+Members were also encouraged to pass the call for information to whomever may be able to provide useful input.
+The responses from the community can be found `here <https://confluence.lsstcorp.org/display/LSSTCOM/2021-05-14+On-the-fly+analysis+for+observers+Meeting+Minutes#id-20210514OntheflyanalysisforobserversMeetingMinutes-On-the-flyAnalysisUse-Cases>`_.
+
+Many of the examples from the community had significant overlap of information.
+To amalgamate the information and create a basis from which to work, a series of standardized use-cases were developed which followed a common format and layout.
+A standardized summary was also created to ease comparisons between use-cases.
+
+The standardized use-cases associated with this deliverable are found at :ref:`pg-D1-Use-cases` page.
+
+.. _Deliverable 2:
+
+Deliverable 2:
+--------------
+
+.. note:: 
+
+   The deliverable description from the `charge`_ is in the note below to ease readability 
+
+   2. A set of requirements to augment, clarify, or remove ambiguity that may occur from looking only at the use-cases.
+      
+      - This is not a formal and all-encompassing LSE-type requirements document.
+
+Like the use-cases, the requirement deliverables have been separated into two different categories, those which are focused on tooling and procedures for on-the-fly analysis and image visualization.
+The two categories are quite separate in nature but not completely independent.
+
+The detailed list of requirements is found on :ref:`pg-D2-Requirements`.
+
+
+.. _Deliverable 3:
+
+Deliverable 3:
+--------------
+
+.. note:: 
+
+   The deliverable description from the `charge`_ is in the note below to ease readability 
+
+   1. A summary of the suite of currently available tools/systems that may help fulfil the required functionality.
+
+      - Report on their status, limitations, and where their current requirements may need to be expanded to cover any increase in scope
+      - Tools which were considered but found to be inadequate must also be reported
+
+This section contains a summary of the tools currently available for use that are applicable to on-the-fly operations.
+What follows is a list of the tools, each with a short description that contain links to pertinent information.
+
+A high-level overview of these tools and computering resources was presented at the `original presentation given to stakeholders <https://docs.google.com/presentation/d/1i4p-sg42FXtEqGVqIZMeFadWSZZ0Lu_CpoqEafkMfy4/edit#slide=id.gd8dafc0d0d_0_30>`_.
+These slides may contain additional information and/or a more detailed description and use-case.
+
+Currently Available Tools
```

Comment:
Not sure if the order of the descriptions of the tools was selected in a particular way. It might make sense to put related tools in sequential order.

Reply:
I tried to do that... but honestly it's not really a trivial thing to do as it's somewhat use-case (and even person) specific.
Do you  have a suggestion on the order?